### ISSUE TITLE
USA Chinook model issues

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#1466]
- [ ] 1. The rotor blades spin too slowly

### 1. Slow rotor blades

https://user-images.githubusercontent.com/4720891/201539607-a3ede4f3-a122-41bd-b5e2-c509e51ddc72.mp4

**Comment by commy2** [comment]
> The rotor blades spin too slowly

No they don't.

https://en.wikipedia.org/wiki/Wagon-wheel_effect

**Comment by ExiLe-ZH** [comment]
> > The rotor blades spin too slowly
> 
> 
> 
> No they don't.
> 
> 
> 
> https://en.wikipedia.org/wiki/Wagon-wheel_effect

Yup probably looks different with different screen refresh rates

**Comment by xezon** [comment]
It definitely rotates much slower than Helix and Comanche. I looked very carefully.
